### DISCUSSION TITLE
Shared Element Support in fabric

### DISCUSSION THREAD

**Discussion by Helmanso** [software-mansion/react-native-reanimated#6410] [Ideas]
Hello, since react native moved officially to fabric, shared element is a very important feature to make smooth & user-friendly apps,
currently the website states  that
Plans for future: Support for the new React Native architecture (Fabric)
any news regarding that matter?

**Comment by MatiPl01** [comment]
Hey!
Shared Element Transitions are on our roadmap and will be added on Fabric but we currently have other priorities, so I can't tell you the expected release date. It might take some time, so stay tuned.

**Reply by allthetime** [reply]
@MatiPl01 hi! Any movement on this since? Is there a better way to keep up with advancements?

**Reply by MatiPl01** [reply]
Unfortunately not. We have other high priority tasks, mostly new-arch related issues. We are planning to add them within the next few months, so stay tuned.

**Reply by allthetime** [reply]
Thanks for the reply - and the amazing library. Good luck!

**Reply by MaxiWainberg** [reply]
Hi, just checking if there is any news on this one 
Thanks :)

**Reply by MatiPl01** [reply]
Not yet. I know that many people are waiting for this feature, but, unfortunately, we had no time to implement it yet.

**Reply by bglgwyng** [reply]
What about opening a bounty for this, using Polar for instance? I want to donate to this.

**Reply by Spring-Rebirth** [reply]
@MatiPl01 Hi! Any movement on this since? It's been 7 months since you last spoke.

**Reply by MaxiWainberg** [reply]
Hi, any news on this one will be appreciated

**Reply by Spring-Rebirth** [reply]
I guess this week or next week will publish new version with this feature.

**Reply by MatiPl01** [reply]
Hey guys! There is an [open PR](https://github.com/software-mansion/react-native-reanimated/pull/7466) that was marked as ready for review a few days ago. The next Reanimated version with Shared Element Transitions support (for now hidden behind a feature flag) should be released soon after it is merged. The review process could take some time, though, as the PR is quite large and we want to review it carefully.

**Reply by MaxiWainberg** [reply]
@Spring-Rebirth @MatiPl01 Thanks, we all waiting for this feature :)

**Reply by mmaxiw** [reply]
Hi, any news on this? i mean, it will be ready for production anytime soon?
Thanks and Thank you for your hard work!

**Reply by mmaxiw** [reply]
Hi, any news on this one?
